src/utils/Libs/requestUrls.ts

```diff
@@ -1,3 +1,23 @@
+export const MemberController = {
+	signin: () => {
+		return '/signin';
+	},
+
+	signup: () => {
+		return '/signup';
+	},
+
+	auth: () => {
+		return '/signup/email';
+	},
+
+	authcheck: () => {
+		return '/signup/email/check';
+	},
+}
```

Comment:
이거는 이렇게 하는것보다는 
```suggestion
	signin: '/signin',
	signup: '/signup',
	auth: '/signup/email',
	authcheck: '/signup/email/check',
}
```
이렇게 하고
변수가 필요한 경우에는


ex)

```ts
board(boardIdx) { return `/board/${boardIdx}`}
```

BoardController.board(boardIdx)
이런식으로 접근하는것도 괜찮을거 같아